examples/src/main/scala/org/catapult/sa/fulgurite/examples/NDVI.scala

```diff
@@ -0,0 +1,51 @@
+package org.catapult.sa.fulgurite.examples
+
+import com.github.jaiimageio.plugins.tiff.BaselineTIFFTagSet
+import org.catapult.sa.fulgurite.geotiff.{GeoTiffMeta, Index}
+import org.catapult.sa.fulgurite.spark.{Argument, Arguments, GeoSparkUtils}
+
+/**
+  * Simple example converting a three colour GeoTIFF into a single band black and white image.
+  */
+object NDVI extends Arguments {
+  override def allowedArgs(): List[Argument] = InputOutputArguments
+
+  def main(args : Array[String]) : Unit = {
+    val opts = processArgs(args)
+    val sc = getSparkContext("Example-BW", "local[3]")
+    val metaData = GeoTiffMeta(opts("input"))
+    val inut = "input"
+      println(inut)
+
+    if (metaData.samplesPerPixel == 1) {
+      println("requires a GeoTIFF with more than one band of data")
+    }
+
+    val input = GeoSparkUtils.GeoTiffRDD(opts("input"), metaData, sc, 4000000)
+      .filter(e => e._1.band==2 || e._1.band==3)//assuming e starts from 0, getting bands 3 and 4 from the array, prod

```

Comment:
Can these band numbers be pulled out into arguments or just constants? they are used in a few places, it would be nicer if we had red or nir instead of 2 and 3 as it is more descriptive to people who come along later.